lib/components/input-group-addon.vue

```diff
@@ -0,0 +1,17 @@
+
+<template>
+    <span class="input-group-addon" :id="id">
+        <slot></slot>
+    </span>
+</template>
+
+<script>
+    export default {
+        props: {
+            id: {
+                type: String,
+                default: 'b_input_geoup_addon_' + this._uid
```

Comment:
I think this won't work correctly as `this` is not available in props. would be really need that id?